tests/integration/framework-detection.test.js

```diff
@@ -10,6 +10,12 @@ import { normalize } from './utils/snapshots.js'
 
 const content = 'Hello World!'
 
+// Normalize random ports
+const normalizeSnapshot = (output, opts) =>
+  normalize(output, opts)
+    .replaceAll(/localhost:\d+/g, 'localhost:88888')
+    .replaceAll(/listening to \d+/g, 'listening to 88888')
```

Comment:
Not critical, but: I would change this to `{PORT_NORMALIZED}` to make it completely clear to anyone viewing the output/snapshot that we normalize this. Using an out-of-range value is a hint but I think it'd be kinder to be more explicit.

Reply:
Good idea! Pushing.